build_scripts/build_cmake.sh

```diff
@@ -16,7 +16,7 @@
 
 # CMake
 pushd third_party/CMake
-./bootstrap --parallel=$(grep ^processor /proc/cpuinfo | wc -l)
+./bootstrap --parallel=$(grep ^processor /proc/cpuinfo | wc -l) --prefix=${INSTALL_PREFIX}
```

Comment:
The same problem as with protobuf applies - it should be installed in host, not target architecture destination directory.
Maybe there should be one more variable - `HOST_INSTALL_PREFIX`?

Reply:
Fixed in 3030b7f3b127f281bca2bc91fb37207a4afcd2a6